test/binaryen.js/expressionrunner.js

```diff
@@ -5,6 +5,15 @@ console.log("// ExpressionRunner.Flags.TraverseCalls = " + Flags.TraverseCalls);
 
 binaryen.setAPITracing(true);
 
+function assertDeepEqual(x, y) {
+  if (typeof x === "object") {
+    for (var i in x) assertDeepEqual(x[i], y[i]);
+    for (i in y) assertDeepEqual(x[i], y[i]);
```

Comment:
Can we do `var i` here, too, or would that be unidiomatic or bad? Seeing the variable reused like this gives me the heebie jeebies.